ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -791,6 +798,69 @@ private void SensorMonitor()
                 azTempSafe = checkTemp(azTemp, azTempSafe);
                 elTempSafe = checkTemp(elTemp, elTempSafe);
 
+                // Sensor status routine, checks for each sensor to update the status in the DB
+                // Check Gate
+                SensorStatusEnum gate = SensorStatusEnum.NORMAL;
+
+                // Check azimuth temp 1
+                SensorStatusEnum azTemp1 = SensorStatusEnum.NORMAL;
```

Comment:
I understand we are not adding it in now, but I want to mention that all ESS sensors will be getting their statuses from the RadioTelescope's `SensorNetworkServer`, which is updated automatically, and can be simply assigned here to the correct sensor status.